ports/linux/datetime-init.c

```diff
@@ -85,5 +134,5 @@ bool datetime_local(BACNET_DATE *bdate,
  */
 void datetime_init(void)
 {
-    /* nothing to do */
+    handler_timesync_set_callback_set(&datetime_timesync);
```

Comment:
Although the set of Linux projects that want their clock updated on a BACnet timesync is non-zero, there are probably a non-zero set of them that don't want that feature. 

Let's move this callback set to the server example main.c module, and add an empty dummy stub function of datetime_timesync() to the MacOS and Win32 ports datetime-init.c module.